### PR TITLE
Feature/fsar 24 document downdload

### PR DESCRIPTION
**Pull request by Yinuo-M**
- Create storybook component
- Create article two column layout and article page
- Add the component to the article page

### CODE REVIEW

File: src/components/general/DocumentDownload/documentDownload.stories.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  new_window: 'Open in a new window',
+  size: '559.42 KB',
+};
+
+export const WithRegion = Template.bind({});
+WithRegion.args = {
+  ...WithoutRegion.args,
+  region: ['England'],
+  format: 'pdf',
+};
```
Comment (c-ancia) [inline src/components/general/DocumentDownload/documentDownload.stories.js:26]: Can you add an example with multiple regions please?

File: src/index.js [hunk 2]
```diff
@@ -43,6 +44,7 @@ import './components/landing/ServiceLinks/serviceLinks';
 import './layout/formLayout/formLayout';
 import './layout/twoColumn/twoColumn';
 import './layout/fullWidth/fullWidth';
+import './layout/articleTwoColumn/articleTwoColumn';
```
Comment (c-ancia) [inline src/index.js:47]: I'm picky here but I think I'd name it ContentLayout. So can you change the whole naming accordingly please?

**Review by c-ancia — CHANGES_REQUESTED** [review]

File: src/components/general/DocumentDownload/documentDownload.html.twig [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{% set region_is_defined = region is defined and region is not empty %}
+{% if region_is_defined %}
+{% endif %}
```
[the remark this answers is not in the document]
Comment (Yinuo-M) [reply src/components/general/DocumentDownload/documentDownload.html.twig:3]: Yes I've moved endif

File: src/components/general/DocumentDownload/documentDownload.html.twig [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+{% if region|length == 1 %}
+  {% set region_class = "document-download--" ~ region[0]|lower|replace({" ": "-"}) %}
+  {% set region_string = region[0] %}
+{% elseif region|length > 1 %}
+  {% set region_class = "document-download--multi" %}
+  {% set region_first = region|slice(0, region|length - 1) %}
```
[the remark this answers is not in the document]
Comment (Yinuo-M) [reply src/components/general/DocumentDownload/documentDownload.html.twig:9]: No, the second argument of slice specifies how many items you want to include, not an index

File: src/components/general/DocumentDownload/documentDownload.html.twig [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+<section class="document-download {{ region_class is defined ? region_class : "" }}">
+  {% if region_is_defined %}
+    <span class="document-download__region">{{ region_string }}</span>
+  {% endif %}
+  <div class="document-download__content document-download__content--{{ region_is_defined ? "with-region" : "" }}">
+    <span class="document-download__format document-download__format--{{ format|lower }}" aria-hidden="true">{{ format|upper }}</span>
+    <a href="{{ link }}" class="document-download__link" target="_blank">
+      <span class="visually-hidden">{{ view }} </span>
+      {{ name }}
+      <span class="visually-hidden"> {{ as }} {{ format }}({{ new_window }})</span>
```
[the remark this answers is not in the document]
Comment (Yinuo-M) [reply src/components/general/DocumentDownload/documentDownload.html.twig:22]: So the "view" and "as format new window" are there for screenreaders to read out. E.g. if it's a PDF document called "Cats", screenreaders will say 'view Cats as PDF in a new window', which makes more sense and provides more context for visually impaired users. I copied this technique from the [live fsa site ](http://foodgovukstg.prod.acquia-sites.com/business-guidance/applying-for-approval-of-a-meat-establishment)

File: src/components/general/DocumentDownload/documentDownload.stories.js (hunk 1 above)
Comment (Yinuo-M) [reply src/components/general/DocumentDownload/documentDownload.stories.js:26]: done

File: src/index.js (hunk 2 above)
Comment (Yinuo-M) [reply src/index.js]: Done

File: src/components/general/DocumentDownload/documentDownload.html.twig (hunk 5 above)
[the remark this answers is not in the document]
Comment (c-ancia) [reply src/components/general/DocumentDownload/documentDownload.html.twig:22]: Yes, I just meant in the new design, it seems Rachael wants "as PDF" to be visible
https://app.zeplin.io/project/5c6fadbfbc13ad1a90fbd4a8/screen/6151add3489aab13acb225ee

**Pull request merged**